Sourcecode/private/mxtest/api/ApiMuAccidentals1ScoreData.h

```diff
@@ -91,8 +91,19 @@ namespace mxtest
         measure->timeSignature.beatType = 4;
         measure->timeSignature.isImplicit = true;
         measure->staves.emplace_back( StaffData{} );
+        measure->keys.emplace_back( KeyData{} );
```

Comment:
OK, I tried adding a test. But this doesn't do anything. I found 2 test files which test keys, this and ApiLy43eScoreData. I don't get what this tests.

Reply:
Just add a new test for this `api/KeyDataTest.cpp`.

ApiLy43eScoreData is probably related to a test file by that name from the Lilypond MusicXML test suite.